install-dev/upgrade/sql/1.7.5.0.sql

```diff
@@ -64,7 +64,7 @@ ALTER TABLE `PREFIX_order_detail`
   CHANGE `product_reference` `product_reference` varchar(64) DEFAULT NULL;
```

Comment:
in TABLE `PREFIX_order_detail` , product_supplier_reference need to be change to varchar(64) too, no?

Reply:
Nice find @BenTen, thanks!